hash.c

```diff
@@ -3576,21 +3576,22 @@ rb_hash_to_h_block(VALUE hash)
 
 /*
  *  call-seq:
- *    to_h -> self or new_hash
  *    to_h {|key, value| ... } -> new_hash
+ *    to_h -> self or new_hash
  *
- *  For an instance of +Hash+, returns +self+.
- *
- *  For a subclass of +Hash+, returns a new +Hash+
- *  containing the content of +self+.
+ *  With a block is given, returns a new hash whose content is based on the block;
```

Comment:
"With a block is given" typo?

Reply:
Ook!  Thanks, @n-rodriguez.  It's good to see that someone's reading what I write.

Reply:
Fixed.